cumulus/tasks/queue-pdrs/tests/index.js

```diff
@@ -1,53 +1,170 @@
+
+  await validateOutput(t, output);
+  t.deepEqual(output, { pdrs_queued: 2 });
+});
+
+test('The correct output is returned when no PDRs are queued', async (t) => {
+  const event = t.context.event;
+  event.input.pdrs = [];
+
+  await validateConfig(t, event.config);
+  await validateInput(t, event.input);
 
-  t.context.bucket = randomString();
-  return s3().createBucket({ Bucket: t.context.bucket }).promise();
+  const output = await queuePdrs(event);
+
+  await validateOutput(t, output);
+  t.deepEqual(output, { pdrs_queued: 0 });
 });
 
-test.afterEach.always((t) =>
-  Promise.all([
-    recursivelyDeleteS3Bucket(t.context.bucket),
-    sqs().deleteQueue({ QueueUrl: t.context.queueUrl }).promise()
-  ]));
+test('PDRs are added to the queue', async (t) => {
+  const event = t.context.event;
+  event.input.pdrs = [
+    { name: randomString(), path: randomString() },
+    { name: randomString(), path: randomString() }
+  ];
+
+  await validateConfig(t, event.config);
+  await validateInput(t, event.input);
 
-test('queue pdrs', async (t) => {
-  const Bucket = t.context.bucket;
-  const ParsePdrTemplate = `s3://${Bucket}/dev/workflows/ParsePdr.json`;
+  const output = await queuePdrs(event);
 
-  await aws.s3().putObject({
-    Bucket,
-    Key: 'dev/workflows/ParsePdr.json',
-    Body: JSON.stringify(workflowTemplate)
+  await validateOutput(t, output);
+
+  // Get messages from the queue
+  const receiveMessageResponse = await sqs().receiveMessage({
+    QueueUrl: t.context.event.config.queueUrl,
+    MaxNumberOfMessages: 10,
+    WaitTimeSeconds: 1
   }).promise();
+  const messages = receiveMessageResponse.Messages;
 
-  MockAWS.stub('StepFunctions', 'describeExecution').returns({
-    promise: () => Promise.resolve({})
-  });
+  t.is(messages.length, 2);
+});
+
+test('The correct message is enqueued', async (t) => {
+  const event = t.context.event;
+  event.input.pdrs = [
+    {
+      name: randomString(),
+      path: randomString()
+    },
+    {
+      name: randomString(),
+      path: randomString()
+    },
+  ];
+
+  await validateConfig(t, event.config);
+  await validateInput(t, event.input);
+
+  const output = await queuePdrs(event);
 
-  const input = Object.assign({}, inputJSON);
-  input.config.templateUri = ParsePdrTemplate;
-  input.config.queueUrl = t.context.queueUrl;
+  await validateOutput(t, output);
 
-  return handler(input, {}, (e, output) => {
-    t.ifError(e);
-    t.is(typeof output, 'object');
-    t.is(output.pdrs_queued, 2);
-    MockAWS.StepFunctions.restore();
+  // Get messages from the queue
+  const receiveMessageResponse = await sqs().receiveMessage({
+    QueueUrl: t.context.event.config.queueUrl,
+    MaxNumberOfMessages: 10,
+    WaitTimeSeconds: 1
+  }).promise();
+  const messages = receiveMessageResponse.Messages.map((message) => JSON.parse(message.Body));
+
+  t.is(messages.length, 2);
+
+  const receivedPdrnames = messages.map((message) => message.payload.pdr.name);
```

Comment:
Nit: (Here and elsewhere in this pr)
> If your function takes a single argument and doesn’t use braces, omit the parentheses

https://github.com/airbnb/javascript#arrows--one-arg-parens

But this is not consistent across the repo, so I don't feel strongly about leaving as is.

Reply:
Airbnb's guide says to omit the braces, but our project style guide overrides that:

https://github.com/cumulus-nasa/cumulus/blob/master/.eslintrc.json#L95